themes/default/content/registry/packages/azure-native-v2/installation-configuration.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Azure Native (v2 Preview) Installation & Configuration
+meta_desc: How to set up credentials to use the Pulumi Azure Native Provider and choose configuration options to tailor the provider to suit your use case.
+layout: installation
+---
+
+To provision resources with the Pulumi Azure Native provider, you need to have Azure credentials.
+Your Azure credentials are never sent to Pulumi.com.
+Pulumi uses the Azure SDK and the credentials in your environment to authenticate requests from your computer to Azure.
+
+## Installation
+
+The Azure Native provider is available as a package in all Pulumi languages:
+
+* JavaScript/TypeScript: [`@pulumi/azure-native`](https://www.npmjs.com/package/@pulumi/azure-native/v/2.0.0-alpha.1)
+* Python: [`pulumi-azure-native`](https://pypi.org/project/pulumi-azure-native/2.0.0a1/)
+* Go: [`github.com/pulumi/pulumi-azure-native/sdk/go/azure`](https://github.com/pulumi/pulumi-azure-native/releases/tag/v2.0.0-alpha.1)
+* .NET: [`Pulumi.AzureNative`](https://www.nuget.org/packages/Pulumi.AzureNative/2.0.0-alpha.1)
+* Java: `com.pulumi.azurenative` [Pre-release unavailable]
```

Comment:
@mnlumi, thoughts on removing this line entirely?

Reply:
@sean1588 we dont have to block merging on this comment

Reply:
I was debating removing it, but didn't want anyone to infer that Java would never be available for v2 so I left it and called out that the pre-release is unavailable. I imagine people that use Java would expect this.